### PR TITLE
Small UI tweak to side navigation

### PR DESCRIPTION
**Pull request by hursey013**
Anchors caret to the last word of section name in side navigation to avoid the icon breaking onto it's own line.

Before:
screen shot 2018-08-14 at 4 25 19 pm
screen shot 2018-08-14 at 4 26 41 pm

After:
screen shot 2018-08-14 at 4 23 40 pm
screen shot 2018-08-14 at 4 24 23 pm

### CODE REVIEW

File: src/components/Navigation/ToggleItem.jsx [hunk 1]
```diff
@@ -40,14 +40,19 @@ class ToggleItem extends React.Component {
   render() {
     const url = this.url()
     const active = this.isActive()
+    const sectionName = this.props.section.name.split(' ')
+    const lastWord = sectionName.pop()
```
Comment (afeld) [inline src/components/Navigation/ToggleItem.jsx]: Mind adding a comment about why this is being done?

**Comment by hursey013** [comment]
In hindsight I realized this could be done with just CSS so removed the additional javascript and as an added bonus, removed some unnecessary HTML markup. The visual output still remains the same as the screenshots above.

**Comment by afeld** [comment]
Is the `App` snapshot up-to-date? It shouldn't have changed that much...

You may need to add the [workaround for the generated field GUIDs](https://github.com/18F/e-QIP-prototype/blob/1ef519bab9d31f5c26b002003bb01d24247e66ab/src/components/Section/Identification/Identification.test.jsx#L19-L25). Trying out a better way to handle that across the board...

**Comment by hursey013** [comment]
@afeld yeah I'm not sure what's up with the `App` snapshot, it doesn't seem relevant to my changes so perhaps it's out of date and was not previously committed - is that a possibility?

**Comment by afeld** [comment]
Ahh, I see the problem. Pull request coming.

**Pull request merged**